file_name_in: "/AtestCollisionAvoidance/box_db"
file_name_out: "/AtestCollisionAvoidance/result_ca/"
file_name_hand_point: "/hand_point/hand_points"